ansible/ansible-content-parser
# Include sage-scan in version outputs

Issue: #30

## Description


Include the version string of `sage-scan` to the outputs displayed when `ansible-content-parser` is invoked with the `--version` option. Also updated the dependencies so that `sage-scan >= 0.0.3` will be used (for #30).

```
 $ ansible-content-parser --version
ansible-content-parser 0.0.8 using ansible-lint:6.22.1 ansible-core:2.16.1 sage-scan:0.0.3
```

## Testing



For issue #30:
1. Build and install ansible-content-parser using [this instruction](https://github.com/ansible/ansible-content-parser/wiki/Build)
1. Run the following command and make sure `ftdata.json` is created in `out_dir`.  Note that there are some error messages displayed for Ansible tasks w/o names. You can ignore those errors.
    ```
    ansible-content-parser --skip-ansible-lint https://github.com/ansible/ansible-examples out_dir
    ```
For the change for `--version` option, execute `ansible-content-parser --version` and verify the version of `sage-scan` shows up in the outputs.

### Steps to test

See the Testing section.

### Scenarios tested


See the Testing section.  For the version output, a unit test is also updated.
